### PR TITLE
Refreshed site look and contents

### PR DESCRIPTION
**Pull request by vogievetsky**
Check out a preview version here: https://staging-druid.imply.io

New website uses SASS to preprocess CSS and removes the defunct blog link.
Many small changes to content and layout.

Most notably:

- Links in the header are different (Download is emphasized)
- More content on the home screen / removed old 'overview page'
- New 'technology' page to describe how druid works
- Old link to blog removed (blog content is still there)
- The site used to really heavily on [bootstrap](https://getbootstrap.com/) this has been greatly reduced, bootstrap (or what is left of it) is now part of the site assets and has been cleaned using [purifycss](https://github.com/purifycss/purifycss).
- Made https friendly

Main page:

image

Technology:

image

Docs:

image

Download:

image

Small screen (responsive):

image

To run this just do:

```
npm install
gulp
npm run run
```

Or just have a look at https://staging-druid.imply.io/

### CODE REVIEW

File: technology.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  <img src="img/diagram-4.png">
+</div>
+
+## Storage
+
+Like many analytic data stores, Druid stores data in columns. Depending on the type of column (string, number, etc), different compression and encoding methods are applied. Druid also builds different types of indexes based on column type.
+
+Similar to logsearch systems, Druid builds inverted indexes for string columns for fast search and filter. Similar to timeseries databases, Druid intelligently partitions data by time to enable fast time-oriented queries.
+
+Unlike many traditional systems, Druid can optionally pre-aggregate data as it is ingested. This pre-aggregation step is know as [rollup](link to rollup docs), and can lead to dramatic storage savings.
```
[the remark this answers is not in the document]
Comment (clintropolis) [reply technology.md]: > This pre-aggregation step is _known_ as ...

**Comment by gianm** [comment]
We've posted a version of this site on https://staging-druid.imply.io/ to make it easier to review.

File: index.html [hunk 2]
```diff
@@ -10,10 +10,10 @@
 <main class="druid-masthead">
   <div class="container">
   <div class="row">
-    <div class="col-md-8 col-md-offset-2 text-center">
-    <p class="lead"><b>Druid</b> is a high-performance, column-oriented, distributed data store.</p>
-    <p><a class="btn btn-lg" href="downloads.html"><span class="fa fa-download"></span> Download</a>
-    <a class="btn btn-lg" href="https://github.com/druid-io/druid/"><span class="fa fa-github"></span> GitHub</a></p>
+    <div class="text-center">
+    <p class="lead">Apache Druid (incubating) is a high performance analytics data store for event-driven data.</p>
```
[the remark this answers is not in the document]
Comment (fjy) [reply index.html]: It is at the bottom of hte page

File: downloads.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Get Started
+
+To get started with Druid, visit the [quickstart](/docs/latest/tutorials/quickstart.html).
+
+To learn more about releases, please visit the [versioning page](/docs/latest/development/versioning.html).
+
+## Optional Dependencies
+
+### Metadata store
+Due to licensing, we've separated the metadata store extension from main Druid release. If you are deploying a distributed cluster and would like to use an external metadata store, please untar these tarball and follow [these steps](/docs/latest/operations/including-extensions.html).
```
[the remark this answers is not in the document]
Comment (vogievetsky) [reply downloads.md]: I tried to reword this.

File: index.html [hunk 4]
```diff
@@ -23,79 +23,147 @@
...
+          <h5>Visualize</h5>
+          <p>
+            Visualize data in Druid with <a href='https://github.com/implydata/pivot'>Pivot</a> and <a href='https://github.com/apache/incubator-superset'>Superset</a>.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-question-circle fa"></span>
+          <h5>FAQ</h5>
+          <p>
+            Learn about some of the <a href='faq'>most common questions about Druid</a>.
```
[the remark this answers is not in the document]
Comment (vogievetsky) [reply index.html]: That sounds cool, but also the header should be minimalistic. I will think about it.

File: index.html [hunk 5]
```diff
@@ -23,79 +23,149 @@
...
+          <h5>Flexible schemas</h5>
+          <p>
+            Druid gracefully handles evolving schemas and <a href="/docs/latest/ingestion/flatten-json">nested data</a>.
+          </p>
+        </div>
+        <div class="feature">
+          <span class="fa fa-clock fa"></span>
+          <h5>Time-optimized partitioning</h5>
+          <p>
+            Druid intelligently partitions data based on time and time-based queries are significantly faster than traditional databases.
```
Comment (gianm) [inline index.html]: What do you think about removing "time optimized partitioning" (it's cool but I'm not sure it's top 7 features) and adding instead this one:

- *Easy to operate.* Scale up or down by just adding or removing servers, and Druid automatically rebalances. Fault-tolerant architecture routes around server failures.

And link "fault tolerant architecture" to the architecting for failure blog post.

Comment (fjy) [reply index.html]: I believe time optimized partitioning is important

I think this bullet point should be repeated on the technology page

**Pull request merged**